client/app/services/navigateTo.js

```diff
@@ -10,3 +10,13 @@ export default function navigateTo(url, replace = false) {
     $rootScope.$applyAsync();
   }
 }
+
+export function handleNavigationEvent(event, url, replace = false) {
+  event.preventDefault();
+  event.stopPropagation();
+  if (event.ctrlKey || event.metaKey) {
```

Comment:
I've never liked this sort of workaround to adapt `onClick` events to work as `href`. I.e.: I mostly use the middle mouse button for this purpose. However, I also don't have a better option for such cases :man_shrugging:

Reply:
Probably better solution would be to turn table rows into links somehow, but I have no idea how to do that...

Reply:
_(well, I know how, but it requires significant changes to markup)_